trunk/SUAVE/Methods/Propulsion/propeller_design.py

```diff
@@ -165,41 +143,50 @@ def propeller_design(prop,number_of_stations=20):
+        # Step 4, determine epsilon and alpha from airfoil data  
+        if airfoil_flag:   
+            # compute airfoil polars for airfoils 
+            airfoil_polars  = compute_airfoil_polars(a_geo, a_pol)  
+            airfoil_cl_surs = airfoil_polars.lift_coefficient_surrogates 
+            airfoil_cd_surs = airfoil_polars.drag_coefficient_surrogates 
             
-            # Newton Raphson Iteration 
-            while cl_diff > 1E-3:
-                
-                Cl_guess       = airfoil_cl_surs[a_geo[a_loc[i]]](RE[i],AoA_old_guess,grid=False) - Cl 
-                
-                # central difference derivative 
-                dx             = 1E-5
-                dCL            = (airfoil_cl_surs[a_geo[a_loc[i]]](RE[i],AoA_old_guess + dx,grid=False) - airfoil_cl_surs[a_geo[a_loc[i]]](RE[i],AoA_old_guess- dx,grid=False))/ (2*dx)
-                 
-                # update AoA guess 
-                AoA_new_guess  = AoA_old_guess - Cl_guess/dCL
-                AoA_old_guess  = AoA_new_guess 
-                
-                # compute difference for tolerance check
-                cl_diff        = abs(Cl_guess)      
-                 
-                ii+=1 	
-                if ii>10000:	
-                    # maximum iterations is 10000
-                    print('Propeller/Rotor section is not converging to solution')
-                    broke = True	
-                    break                    
+            alpha = np.zeros_like(RE)
+            Cdval = np.zeros_like(RE)  
+            for i in range(N):
+                AoA_old_guess = 0.01
+                cl_diff       = 1  
+                broke         = False   
+                ii            = 0
                 
-            alpha[i] = AoA_old_guess     
-            Cdval[i] = airfoil_cd_surs[a_geo[a_loc[i]]](RE[i],alpha[i],grid=False)  
-
+                # Newton Raphson Iteration 
+                while cl_diff > 1E-3:
+                    
+                    Cl_guess       = airfoil_cl_surs[a_geo[a_loc[i]]](RE[i],AoA_old_guess,grid=False) - Cl 
+                    
+                    # central difference derivative 
+                    dx             = 1E-5
+                    dCL            = (airfoil_cl_surs[a_geo[a_loc[i]]](RE[i],AoA_old_guess + dx,grid=False) - airfoil_cl_surs[a_geo[a_loc[i]]](RE[i],AoA_old_guess- dx,grid=False))/ (2*dx)
+                     
+                    # update AoA guess 
+                    AoA_new_guess  = AoA_old_guess - Cl_guess/dCL
+                    AoA_old_guess  = AoA_new_guess 
+                    
+                    # compute difference for tolerance check
+                    cl_diff        = abs(Cl_guess)      
+                     
+                    ii+=1 	
+                    if ii>10000:	
+                        # maximum iterations is 10000
+                        print('Propeller/Rotor section is not converging to solution')
+                        broke = True	
```

Comment:
I realize you copied this from the propeller script, you don't need broke in this case since you're not saving that output for later

Reply:
Got it, removed.